R/insertPackage.R

```diff
@@ -130,7 +130,7 @@ identifyPackageType <- function(file) {
     ret <- if (grepl("_.*\\.tar\\..*$", file)) {
         "source"
     } else if (grepl("_.*\\.tgz$", file)) {
-        "mac.binary"
+        "mac.binary.mavericks"
```

Comment:
That doesn't look right.  As long as Simon supports both flavors, shouldn't we too?